src/shared/lib/mappers/company.js

```diff
@@ -0,0 +1,6 @@
+'use strict';
+
+const mapCompanyToString = company =>
```

Comment:
Is this required when you only getting the name from the company?

Reply:
Even though this logic is very simple at the moment, I think it is best to have the logic in one central location where it can easily be modified rather than repeated around the project